pxtarget.json

```diff
@@ -605,10 +605,10 @@
         },
         "blocklyOptions": {
             "grid": {
-                "spacing": 20,
-                "length": 100,
-                "colour": "rgba(107, 79, 118, 0.10)",
-                "snap": true
+                "spacing": 30,
+                "length": 1,
```

Comment:
I don't know exactly what's happening here, but is 1 a typo? That seems too small to be useful.

Reply:
Nope, not a typo! 1 makes it so that the point that is being used for the grid is a circle! https://developers.google.com/blockly/guides/configure/web/grid#length 

I guess I didn't read the docs close enough, though, because it looks like 1 is the default, so I might be able to just get rid of that field